docs/release_notes/sat_2.2_release_notes.md

```diff
@@ -20,46 +20,48 @@ The following sections detail the changes in this release.
 ### `sat` Command Unavailable in `sat bash` Shell
 
 After launching a shell within the SAT container with `sat bash`, the `sat`
-command will not be found. For example:
+command will not be found.
 
-```screen
-(CONTAINER-ID) sat-container:~ # sat status
+(`(CONTAINER-ID) sat-container:~ #`) Here is an example output after running `sat status`:
+
+```bash
 bash: sat: command not found
 ```
 
-This can be resolved temporarily in one of two ways. `/sat/venv/bin/` may be
+(`(CONTAINER-ID) sat-container:~ #`) This can be resolved temporarily in one of two ways. `/sat/venv/bin/` may be
 prepended to the `$PATH` environment variable:
 
-```screen
-(CONTAINER-ID) sat-container:~ # export PATH=/sat/venv/bin:$PATH
-(CONTAINER-ID) sat-container:~ # sat status
+```bash
+export PATH=/sat/venv/bin:$PATH
+sat status
 ```
 
-Or, the file `/sat/venv/bin/activate` may be sourced:
+(`(CONTAINER-ID) sat-container:~ #`) You can also source the file `/sat/venv/bin/activate`:
 
-```screen
-(CONTAINER-ID) sat-container:~ # source /sat/venv/bin/activate
-(CONTAINER-ID) sat-container:~ # sat status
+```bash
+source /sat/venv/bin/activate
+sat status
 ```
 
 ### Tab Completion Unavailable in `sat bash` Shell
 
 After launching a shell within the SAT container with `sat bash`, tab completion
 for `sat` commands does not work.
 
-This can be resolved temporarily by sourcing the file
+(`(CONTAINER-ID) sat-container:~ #`) This can be resolved temporarily by sourcing the file
 `/etc/bash_completion.d/sat-completion.bash`:
 
-```screen
+```bash
 source /etc/bash_completion.d/sat-completion.bash
 ```
 
 ### OCI Runtime Permission Error when Running `sat` in Root Directory
 
-`sat` commands will not work if the current directory is `/`. For example:
+`sat` commands will not work if the current directory is `/`.
+
+(`ncn-m001:/ #`) Here is an example output after running `sat --help`:
```

Comment:
This one's kinda tricky because you are trying to show that the working directory in `/`, but that's not part of our command-prompt conventions, so I'd still omit that part.

```suggestion
(`ncn-m001#`) Here is an example output after running `sat --help`:
```

Reply:
Updated in next commit.